pantavisor.c

```diff
@@ -974,14 +973,6 @@ int pv_start()
 
 	setrlimit(RLIMIT_CORE, &core_limit);
 
-	pv_paths_storage_file(path, PATH_MAX, COREPV_FNAME);
```

Comment:
so where are they going to be stored now with this patch?

Reply:
whatever is in sysctl.kernel.core_pattern in pantavisor.config ... in the last commit i added a default maybe see what i did.